go.mod

```diff
@@ -15,20 +15,20 @@ require (
 	github.com/aws/aws-sdk-go-v2/feature/s3/manager v1.15.15
 	github.com/aws/aws-sdk-go-v2/service/s3 v1.48.1
 	github.com/containerd/console v1.0.4
-	github.com/containerd/containerd v1.7.23
-	github.com/containerd/containerd/api v1.7.19
+	github.com/containerd/containerd/api v1.8.0
+	github.com/containerd/containerd/v2 v2.0.0
 	github.com/containerd/continuity v0.4.4
-	github.com/containerd/errdefs v0.3.0
-	github.com/containerd/fuse-overlayfs-snapshotter v1.0.8
+	github.com/containerd/errdefs v1.0.0
+	github.com/containerd/fuse-overlayfs-snapshotter/v2 v2.0.0
 	github.com/containerd/go-cni v1.1.10
 	github.com/containerd/go-runc v1.1.0
 	github.com/containerd/log v0.1.0
 	github.com/containerd/nydus-snapshotter v0.14.0
-	github.com/containerd/platforms v0.2.1
-	github.com/containerd/stargz-snapshotter v0.15.1
-	github.com/containerd/stargz-snapshotter/estargz v0.15.1
-	github.com/containerd/typeurl/v2 v2.2.0
-	github.com/containernetworking/plugins v1.4.0
+	github.com/containerd/platforms v1.0.0-rc.0
+	github.com/containerd/stargz-snapshotter v0.15.2-0.20241017154050-a6b9bdb5a9e1
```

Comment:
Just tagged, https://github.com/containerd/stargz-snapshotter/releases/tag/v0.16.0

Reply:
It forced me to bump go to 1.23.0 in `go.mod`. Is it ok?

Reply:
Let me ask if that was intentional, the go.mod version should really be the max min version of go required

Reply:
This should fix it https://github.com/containerd/stargz-snapshotter/pull/1863

Reply:
Thanks, bumbed dep to v0.16.1